test/functional/Java11andUp/src/org/openj9/test/java_lang/Test_String.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2018, 2018 IBM Corp. and others
+ * Copyright (c) 2018, 2019 IBM Corp. and others
```

Comment:
some other tests you could consider are
- what happens when `|n|` is greater or less than the number of characters in your string?
- I think it would be good to have tests for different kinds of strings like the empty string and string without a line ending etc